Sources/Commands/SwiftPackageResolveTool.swift

```diff
@@ -8,32 +8,41 @@
  See http://swift.org/CONTRIBUTORS.txt for Swift project authors
 */
 
+import Basic
 import PackageGraph
 import SourceControl
 
-private class ResolverToolDelegate: DependencyResolverDelegate {
+private class ResolverToolDelegate: DependencyResolverDelegate, CheckoutManagerDelegate {
     typealias Identifier = RepositoryPackageContainer.Identifier
 
     func added(container identifier: Identifier) {
         print("note: considering repository: \(identifier.url)")
     }
+
+    func fetching(handle: CheckoutManager.RepositoryHandle, to path: AbsolutePath) {
+        print("note: fetching \(handle.repository.url) to \(path.asString)")
```

Comment:
Yeah right now we don't have it, at some places I have used OutputByteStream to output status where it was important to test for eg: https://github.com/apple/swift-package-manager/blob/master/Sources/PackageLoading/ModuleMapGenerator.swift#L63 
(though that isn't ideal either for testing)